randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -346,7 +346,7 @@ Extra - room_id: [8]
   * L0 Hatch to Clogged Cavern/Door to Nocturnal Playground
   * Extra - door_idx: (16,)
   > Door to Nocturnal Shaft
-      Trivial
+      Varia Suit and Combat (Intermediate) and Cold Damage ≥ 299
```

Comment:
I thought this was supposed to stay trivial, since you traverse this path in vanilla?

Reply:
I'm leaning towards this stays trivial. 
I could understand that going IN is the vanilla path, and returning on the way OUT is not intended vanilla (given the speedblocks in the next room).
My main point would be that this room becomes a bit confusing in a hypothetical Entrance Rando where all the other connections are considered "trivial" in regards to the blue-X.
![image](https://github.com/randovania/randovania/assets/125271738/7fcfe128-2a2d-48fc-8982-07d8bb6e93f6)

Reply:
this is a fair argument. reverted